app/vmselect/netstorage/tmp_blocks_file.go

```diff
@@ -7,13 +7,18 @@ import (
 	"sync"
 
 	"github.com/VictoriaMetrics/VictoriaMetrics/lib/bytesutil"
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/flagutil"
 	"github.com/VictoriaMetrics/VictoriaMetrics/lib/fs"
 	"github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
 	"github.com/VictoriaMetrics/VictoriaMetrics/lib/memory"
 	"github.com/VictoriaMetrics/VictoriaMetrics/lib/storage"
 	"github.com/VictoriaMetrics/metrics"
 )
 
+var (
+	tmpBufSize = flagutil.NewBytes("search.tmpBufSizeBytes", 0, "In large memory scenarios, controlling the size of temporary buffers to use memory more efficiently")
```

Comment:
```suggestion
	tmpBufSize = flagutil.NewBytes("search.tmpBufSizeBytes", 0, "Size for in-memory data blocks used during processing search requests. By default, the size is automatically calculated based on available memory. Adjust this flag value if you observe that vm_tmp_blocks_max_inmemory_file_size_bytes metric shows much higher values than vm_tmp_blocks_inmemory_file_size_bytes. See https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6851")
```

Reply:
thanks i have update these in #6857.Maybe I should just left the change in CHANELOG.md in this pr?

Reply:
I have just left doc changes in this pr.Plz review later.thanks.:)

Reply:
I think all changes need to be made to cluster branch only. The rest will be cherry-picked by the person who'll merge this PR.

Reply:
Okay, then I think this PR can be closed.The changes made to #6857  are enough